Makefile

```
@@ -290,14 +295,14 @@ lint: export TARGET := default
 lint: ##@test Run code style checks
 	sh scripts/lint-re-frame-in-quo-components.sh && \
```

Maybe we can quiet it down a bit with a `@`?
```suggestion
	@sh scripts/lint-re-frame-in-quo-components.sh && \
```
Same for the other one.